### ISSUE TITLE
data wrapping for single items

### ISSUE THREAD

**Issue by derekphilipau** [robsontenorio/vue-api-query#54]
It is typical for JSON API's to wrap not only multiple results but also single results in a `data` key.  (Even Laravel's API Resources default to this behaviour.)

For multiple results (arrays of items), vue-api-query automagically works regardless of whether or not the `data` key wrapper is present.

I feel that for single results there should be the same behaviour.

Backward-compatibility can be ensured by adding a configuration option or optional function argument to enable this behaviour.  (Similar to `withoutWrapping` in Laravel's API Resources.  See [Eloquent: API Resources](https://laravel.com/docs/5.7/eloquent-resources#data-wrapping))

So, if the backend responds with:
```
{
    "data": {
        id: 1,
        firstname: "John",
        lastname: "Doe",
        age: 25
    }
}
```
And `withWrapping` is set to true, the behaviour should be the same as if the response was:
```
{
    id: 1,
    firstname: "John",
    lastname: "Doe",
    age: 25
}
```

**Comment by peterquentin** [comment]
@derekphilipau I've made a pull request fixing this issue, however I'd say it's a breaking change because right now the single entities are expecting the "data" attribute in the response.

It's also unwrapping nested data attributes, because I need my response to work with vuex-orm.
I hope this helps!

**Comment by robsontenorio** [comment]
@derekphilipau Can you try this branch ?

https://github.com/robsontenorio/vue-api-query/tree/find-wrap

It solves "single item" wrapping issue. Please, tell me if is working as expected.

**Comment by derekphilipau** [comment]
This is fine, I just wonder if it will affect others who are already expecting find() to return a data element, for example I already have code like this:
```
          this.item = TaxCategory.find(this.id).then(response => {
            this.item = new TaxCategory(response.data);
          });
```

**Comment by robsontenorio** [comment]
Maybe a new brand `$find()` would be better in order to avoid breaking changes.

**Comment by derekphilipau** [comment]
Or just changing `find(identifier)` to `find(identifier, isDataWrapped = false)`

**Comment by leeovery** [comment]
I don’t think that breaking change is a big deal. Easier enough to search and replace in a project. I would expect the unwrapping of the outer data element to be the default behaviour, personally :)

**Comment by robsontenorio** [comment]
Agree. find() should be modified. 

If someone can, please submit a PR (including tests) for a new method $find() , just like $get() does.

**Comment by leeovery** [comment]
[https://github.com/robsontenorio/vue-api-query/pull/61](https://github.com/robsontenorio/vue-api-query/pull/61)

**Comment by robsontenorio** [comment]
Solved by #61 . Just released 1.5.0 . Thanks @leeovery !
